Marlin/src/feature/runout.h

```diff
@@ -413,9 +413,6 @@ class FilamentSensorBase {
           const int32_t steps = b->steps.e;
           const float mm = (TEST(b->direction_bits, E_AXIS) ? -steps : steps) * planner.mm_per_step[E_AXIS_N(e)];
           if (e < NUM_RUNOUT_SENSORS) mm_countdown.runout[e] -= mm;
-          #if ENABLED(FILAMENT_SWITCH_AND_MOTION)
-            if (e < NUM_MOTION_SENSORS) mm_countdown.motion[e] -= mm;
```

Comment:
the same code is right above this

Reply:
No, the line right above it is `mm_countdown.runout[e]`: https://github.com/MarlinFirmware/Marlin/blob/83da21089f789033d2a8d5de6433c5fc7cf58b98/Marlin/src/feature/runout.h#L415

The line you're removing has `mm_countdown.motion[e]`: https://github.com/MarlinFirmware/Marlin/blob/83da21089f789033d2a8d5de6433c5fc7cf58b98/Marlin/src/feature/runout.h#L417

Reply:
my mistake